client_secret.json

```diff
@@ -0,0 +1,13 @@
+{
+  "web": {
+    "client_id": "867912232279-88rds6a2r4qfvfp4b631lbfgs46agn1a.apps.googleusercontent.com",
+    "project_id": "bustling-vim-124517",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://accounts.google.com/o/oauth2/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_secret": "xXBYedZfUxKTnWvRR__rOlyz",
+    "javascript_origins": [
```

Comment:
You should never commit secret to Github.
Here you should put variable like: 

GOOGLE_CLIENT_SECRET
and put those value in your local_settings.py file which is ignore.